homeassistant/components/mullvad/translations/en.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
This file should be named `mullvad/strings.json`, as this is a strings source file?
Next run `python3 -m script.translations develop` to generate the resulting en.json file.

Reply:
Thanks for this. I definitely missed this in the documentation. Should be all set now :)